tip-wlan/charts/opensync-gw-cloud/values.yaml

```diff
@@ -11,7 +11,7 @@ replicaCount: 1
 
 image:
   name: opensync-gateway-cloud
-  tag: 0.0.1-SNAPSHOT
+  tag: 1.0.0-SNAPSHOT
```

Comment:
how about defining a single value release-tag:1.0.0-SNAPSHOT and using it everywhere in the charts ?

Reply:
Yes I agree, I will be submitting the changes for that